src/ember-app.js

```diff
@@ -12,6 +13,11 @@ const debug = require('debug')('fastboot:ember-app');
 const FastBootInfo = require('./fastboot-info');
 const Result = require('./result');
 
+const LATEST_SCHEMA_VERSION = '1';
+// this allows fastboot lib to have unpublished features from ember-cli-fastboot but allows
+// people to run integration tests in ember-cli-fastboot
+const NEXT_SCHEMA_VERSION = '2';
```

Comment:
I agree it is cleaner to make it in an integer. Switched to that. However, with the new given scheme of things I don't think we should allow `LATEST_SCHEMA_VERSION++` since it can accidentally cause issues. For example, someone changed the schema version and updated the manifest schema version to 3 while latest version that FastBoot server understands is 2. It will cause breakage.